templates/arc/app/entry.server.tsx

```diff
@@ -39,6 +39,25 @@ export default function handleRequest(
       );
 }
 
+// Allow either isbot@3 or isbot@4 - see https://www.npmjs.com/package/isbot
+function isBotRequest(userAgent: string | null) {
+  if (!userAgent) {
+    return false;
+  }
+
+  // isbot@3
+  if ("default" in isbot && typeof isbot.default === "function") {
+    return isbot.default(userAgent);
+  }
+
+  // isbot@4
+  if ("isbot" in isbot && typeof isbot.isbot === "function") {
+    return isbot.isbot(userAgent);
+  }
+
+  return false;
+}
+
```

Comment:
I guess templates can use v4 as they will only be used when creating new apps?

Reply:
🤣 yeah I realized this too late yesterday and left a post-it note on my desk as I logged off: _"remove isbot changes from templates?"_